lib/use_cases/administrator/check_if_valid_ip.rb

```diff
@@ -13,7 +13,9 @@ def execute(address)
       def valid_address?
         address.present? &&
           address_is_ipv4? &&
-          !(address_is_subnet? || address_allows_all?)
+          !address_is_subnet? &&
```

Comment:
Think we should probably invert & rename these methods, at a scan the `!` can get lost.

Suggest `address_is_not_subnet?`, `address_is_not_wildcard?`, `address_is_not_loopback?`?